tools/network-scene-vis.md

```diff
@@ -0,0 +1,61 @@
+---
+id: netscenevis
+title: Network Scene Visualization
+---
+
+The Network Scene Visualization (NetSceneVis) is a powerful, user-friendly tool included in the Multiplayer Tools package to help you visualize and debug network communication on a per-object basis in the scene view of your project with visualizations such as mesh shading and text overlay.
+
+When a game session is started, network messages contain information about the game state, such as the position of game objects and player actions. The NetSceneVis tool intercepts these network messages to visually display them in the Unity Editor. Each player is represented by a colored sphere, and lines show the network messages being sent between the players. The state of each GameObject in the scene, including their position, rotation, and any other relevant data is also shown.
```

Comment:
I'm concerned that this paragraph doesn't accurately describe the tool 🤔 
1. I think a term like "listen" might be more accurate than "intercept" here, as NetSceneVis is not positioning itself in between the host and client, and does not interfere with or intercept any communication.
2. Each player is _not_ represented as a colored sphere and lines are _not_ drawn to show network messages being sent between players.
3. The tool does not show the state of each game object in the scene, including their position, rotation, and other relevant information

Instead, what the tool does, is to visualize and display bandwidth and ownership on a per object basis using either a textual overlay or mesh shading.